napari/_qt/_qapp_model/_tests/test_view_menu.py

```diff
@@ -50,28 +51,113 @@ def test_toggle_axes_scale_bar_attr(
 
 
 @skip_local_popups
+@pytest.mark.qt_log_level_fail('WARNING')
 def test_toggle_fullscreen(make_napari_viewer, qtbot):
-    """Test toggle fullscreen action."""
+    """
+    Test toggle fullscreen action.
+
+    Check that toggling from a normal and maximized state can be done without
+    generating any type of warning and menu bar elements are visible in any
+    window state.
+    """
     action_id = 'napari.window.view.toggle_fullscreen'
     app = get_app_model()
     viewer = make_napari_viewer(show=True)
 
+    def check_windows_style():
+        if os.name == 'nt':
```

Comment:
```suggestion
        if os.name != 'nt':
            return
```
And dedent rest? It will be more readable.